blockchains/smartchain/assets/0xE94444Ba62208431a27C764E2e28794e81f5B235/info.json

```
@@ -0,0 +1,40 @@
+{
+    "name": "OilMetaGames",
+    "type": "BEP20",
+    "symbol": "OMG",
+    "decimals": "9",
```

Hi @Mazyar60 , this field should be an integer:

```suggestion
    "decimals": 9,
```